sky/templates/kubernetes-sshjump.yml.j2

```diff
@@ -0,0 +1,83 @@
+pod_spec:
+    apiVersion: v1
+    kind: Pod
+    metadata:
+        name: {{ name }}
+        labels:
+            component: {{ name }}
+    spec:
+        serviceAccountName: sshjump
+        volumes:
+          - name: secret-volume
+            secret:
+                secretName: {{ secret }}
+        containers:
+          - name: {{ name }}
+            imagePullPolicy: IfNotPresent
+            image: {{ image }}
+            command: ["python3", "-u", "/skypilot/sky/utils/kubernetes/sshjump-lcm.py"]
+            ports:
+              - containerPort: 22
+            volumeMounts:
+              - name: secret-volume
+                readOnly: true
+                mountPath: /etc/secret-volume
+            lifecycle:
+                postStart:
+                    exec:
+                        command: ["/bin/bash", "-c", "mkdir -p ~/.ssh && cp /etc/secret-volume/ssh-publickey ~/.ssh/authorized_keys && sudo service ssh restart"]
+            env:
+              - name: MY_POD_NAME
+                valueFrom:
+                    fieldRef:
+                        fieldPath: metadata.name
+              - name: MY_POD_NAMESPACE
+                valueFrom:
+                    fieldRef:
+                        fieldPath: metadata.namespace
+              - name: ALERT_THRESHOLD
+                # seconds
+                value: "600"

```

Comment:
@landscapepainter , sshjump monitors the existence of ray pods, every 1 minute (retry_interval). If it sees that no ray pods exist during 10 minutes (alert_threshold) then it removes its service and terminates itself.